README.md

```diff
@@ -24,15 +24,15 @@ There are a million and one great ways to build your next website or application
 This principle of being unopinionated manifests in several important ways including:
 
 * A flat component architecture that lets you compose what you need without struggling with rigid patterns and complex objects.
-* Separating base components and style that supports multiple frameworks without re-writing or duplicating styles. This separation also lets you build your own styles, with your styling technology of choice, without having to re-build or duplicate the base components.
+* Separating base components from styles and design systems to support multiple implementations without re-writing or duplicating styles. Use the design system to customize existing styled components, or build your own styles, with your design system, without having to re-build or duplicate the base components.
```

Comment:
The above comma seems un-necessary but disregard if you disagree.

Reply:
@EisenbergEffect I merge this in so it could go with the other doc work being done over the weekend.  If Chris didn't get a chance to resolve this we can do it in a future PR. I presume we'll be doing a wide edit session on all of this documentation in the future as well.

Reply:
I didn’t see this, but we certainly should do a scrub in the future. For now I’d agree that the most important thing is accurate content. Thx for merging Aaron!